migrations/tenant/0026-files@rename-extension-to-mimetype.sql

```diff
@@ -0,0 +1,12 @@
+DO $$
+BEGIN
+  IF EXISTS (
+    SELECT 1
+    FROM   information_schema.columns
+    WHERE  table_name = 'files'
+    AND    column_name = 'extension'
+  )
```

Comment:
add a statement that check that there is no `mimetype` column

Reply:
done